homeassistant/components/weatherflow/strings.json

```diff
@@ -13,7 +13,8 @@
     },
     "error": {
       "address_in_use": "Unable to open local UDP port 50222.",
-      "cannot_connect": "UDP discovery error."
+      "cannot_connect": "UDP discovery error.",
+      "no_devices_found": "[%key:common::config_flow::abort::no_devices_found%]"
```

Comment:
Should this be a reference to `abort::no_devices_found` or should we add a new key under error in the main `strings.json`. (Reason: The text is exactly the same and it would be a duplicate.)

Reply:
Let's not duplicate so I think this is fine 👍